trino/trino.go

```diff
@@ -132,16 +132,17 @@ const (
 
 	authorizationHeader = "Authorization"
 
-	kerberosEnabledConfig           = "KerberosEnabled"
-	kerberosKeytabPathConfig        = "KerberosKeytabPath"
-	kerberosPrincipalConfig         = "KerberosPrincipal"
-	kerberosRealmConfig             = "KerberosRealm"
-	kerberosConfigPathConfig        = "KerberosConfigPath"
-	kerberosRemoteServiceNameConfig = "KerberosRemoteServiceName"
-	sslCertPathConfig               = "SSLCertPath"
-	sslCertConfig                   = "SSLCert"
-	accessTokenConfig               = "accessToken"
-	explicitPrepareConfig           = "explicitPrepare"
+	kerberosEnabledConfig            = "KerberosEnabled"
+	forwardAuthorizationHeaderConfig = "ForwardAuthorizationHeader"
```

Comment:
Why it's between Kerberos options? Add it at the end.

Reply:
fixed